api/class/Accoutrement

MaximumADHD opened the pull request:
Looking for some feedback on how I documented this, and whether its in a decent enough condition to be merged.

MaximumADHD committed:
Create Accoutrement.md

Anaminus commented:
Falling down a rabbit hole here.

So Accoutrements only behave when the humanoid is alive, among other non-obvious conditions. I think this sort of thing occurs in a variety of classes, so there will likely have to be a concept of a "Living Character Assembly", which includes things like:
> a Model that contains a Humanoid, a BasePart named "Head", and a BasePart named "Torso", in which the Head is attached to the Torso.

Humanoid itself seems to be less strict about the conditions.

Anyway, verifying the behavior of Accoutrements requires clarification on this concept, so I'll only nitpick about document syntax for this first review.

Anaminus committed:
'scuse me while I test this out

- Add top-level summary heading.
- Remove redundant subject in first summary sentence.
- Italics are reserved for parameters.
- Section content should hug the heading.
- Member sections must be ordered alphabetically.

Anaminus reviewed (approved):
Current content is approved as a draft.
